Validation/RecoEgamma/python/egammaValidationMiniAOD_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms

```

Comment:
in fact, I tried to remake the old way when phton and electron where developped on both side (see for example Validation\RecoEgamma\python\egammaValidation_cff.py). 
If you want, I can make it in an independant way.